backend/whatsapp.py

```diff
@@ -182,7 +191,7 @@ def onMessage(self, ws, message):
                     if isinstance(jsonObj, list) and len(jsonObj) > 0:					# check if the result is an array
                         eprint(json.dumps(jsonObj));
                         if jsonObj[0] == "Conn":
-                            Timer(25, lambda: self.activeWs.send('?,,')).start() # Keepalive Request
+                            Timer(20.0, self.keepAlive).start() # Keepalive Request
```

Comment:
👌 with moving the keepalive to `self.keepAlive`
❓ why change the time from 25 to 20?

Reply:
the last keep alive method get disconnect after 40 - 50 secs. and for me each 30 seconds get disconnect . so i set timer on 20